ms/MSOper/MSMetaData.cc

```diff
@@ -3571,9 +3913,22 @@ void MSMetaData::_getScanAndSubScanProperties(
     }
     SHARED_PTR<std::map<SubScanKey, SubScanProperties> > myssprops;
     SHARED_PTR<std::map<ScanKey, ScanProperties> > myscanprops;
+#ifdef _OPENMP
+    if (_parallel) {
+        _computeScanAndSubScanPropertiesParallel(
+            myscanprops, myssprops, showProgress
+        );
+    }
+    else {
+        _computeScanAndSubScanProperties(
+            myscanprops, myssprops, showProgress
+        );
+    }
+#else
     _computeScanAndSubScanProperties(
         myscanprops, myssprops, showProgress
     );
+#endif
```

Comment:
Ah, I was thinking about it incorrectly, so I think you are right there. But there is still a lot of collating of data that is done in the loops following that loop, which isn't necessary to do in the serial-specific version. This may not be all that significant, I'm not sure, but since the serial-specific version already exists, it seems this unnecessary overhead can be avoided simply by using it when necessary.